Makefile

```diff
@@ -185,15 +185,18 @@ else
 	$(INSTALL) -m 0644 $(FBNAME) $(DESTDIR)/$(DATATARGETDIR)/$(FBNAME)
 endif
 
-%.efi: %.so
+%.so.sbat : %.so
```

Comment:
I would rather see this make a .o, have that be linked into the .so file, and place it explicitly after .vendor_cert in the `elf_ARCH_efi.lds` file.  It's also going to need `_sbat` and `_esbat` symbols defined in the `elf_ARCH_efi.lds` file, so we can find it in the self-check.

Reply:
Gotcha. I'll do these changes.